app/src/components/tms/TileMapDetails.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div>
+    <h5>{{ tileMap.title }}</h5>
+    <p v-if="tileMap.abstract" class="text-muted">{{ tileMap.abstract }}</p>
+
+    <h6 class="mt-3">Tile Map Details</h6>
+    <InfoList :info="tileMapInfo" />
+
+    <div v-if="tileMap.tileSets && tileMap.tileSets.length > 0" class="mt-3">
+      <h6>Tile Sets</h6>
+      <div class="d-flex flex-row flex-wrap gap-2 mb-2">
+        <span class="badge bg-secondary"
+          >{{ tileMap.tileSets.length }} available resolutions</span
+        >
+        <span class="badge bg-secondary">{{ tileMap.srs }}</span>
+      </div>
+
+      <table class="table table-sm table-striped">
+        <thead>
+          <tr>
+            <th>Order</th>
+            <th>Units/Pixel</th>
+            <th>Href</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="(tileSet, index) in tileMap.tileSets" :key="index">
+            <td>{{ index + 1 }}</td>
+            <td>{{ tileSet.unitsPerPixel }}</td>
+            <td>
+              <code class="small">{{ tileSet.href }}</code>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+
+    <div v-if="tileMap.boundingBox" class="mt-3">
+      <h6>Bounding Box</h6>
+      <table class="table table-sm">
+        <tr>
+          <th>Min X</th>
+          <td>{{ boundingBoxValues[0] }}</td>
+          <th>Min Y</th>
+          <td>{{ boundingBoxValues[1] }}</td>
+        </tr>
+        <tr>
+          <th>Max X</th>
+          <td>{{ boundingBoxValues[2] }}</td>
+          <th>Max Y</th>
+          <td>{{ boundingBoxValues[3] }}</td>
+        </tr>
+      </table>
+    </div>
+
+    <div v-if="tileMap.metadata && tileMap.metadata.length > 0" class="mt-3">
+      <h6>Metadata</h6>
+      <table class="table table-sm table-striped">
+        <thead>
+          <tr>
+            <th>Type</th>
+            <th>MIME Type</th>
+            <th>Link</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="(item, index) in tileMap.metadata" :key="index">
+            <td>{{ item.type }}</td>
+            <td>{{ item.mimeType }}</td>
+            <td>
+              <a :href="item.href" target="_blank" class="small">{{
+                item.href
+              }}</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</template>
+
+<script>
+import InfoList from '../presentation/InfoList.vue';
+
+export default {
+  name: 'TileMapDetails',
+  components: { InfoList },
+  props: {
+    tileMap: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    boundingBoxValues() {
+      return Array.isArray(this.tileMap.boundingBox)
+        ? this.tileMap.boundingBox
+        : [
+            this.tileMap.boundingBox?.minx,
+            this.tileMap.boundingBox?.miny,
+            this.tileMap.boundingBox?.maxx,
+            this.tileMap.boundingBox?.maxy,
+          ];
```

Comment:
Is that something that application code would be expected to do? IT sounds like the Bbox should always be an array